### PR TITLE
Nwd nnw

### PR DESCRIPTION
**Pull request by ploWoj**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @ploWoj

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ploWoj for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,21 @@
 #pragma once
+#include <cstdlib>
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    int temp;
+    while(rhs!=0){
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:6]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+	    temp=rhs;
+	    rhs=lhs%rhs;
+	    lhs=temp;
+    }
+    return  abs(lhs);
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:11]: One space is redundant

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+        
+    if(lhs == 0 || rhs == 0){
+        return 0;
+    }
+    return abs(lhs*rhs/NWD(lhs,rhs));
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:20]: space

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**